data_psi.go

```diff
@@ -262,46 +265,55 @@ func hasCRC32(tableType string) bool {
 // psiTableType returns the psi table type based on the table id
 // Page: 28 | https://www.dvb.org/resources/public/standards/a38_dvb-si_specification.pdf
 // (barbashov) the link above can be broken, alternative: https://dvb.org/wp-content/uploads/2019/12/a038_tm1217r37_en300468v1_17_1_-_rev-134_-_si_specification.pdf
-func psiTableType(tableID int) string {
+func (t PSITableTypeId) String() string {
```

Comment:
I'd rather call this function `Type` and if you need to implement the `Stringer` interface, you can create a `String` method that calls `Type`. Otherwise I've the feeling this `String` method is not really self-explanatory

Reply:
Sure. I've done the same with `PESIsVideoStreamId` also

Reply:
Fair enough. Renamed.